src/tests/ftest/util/file_count_test_base.py

```diff
@@ -74,7 +78,7 @@ def run_file_count(self):
                     self.log.info("=======>>>Starting MDTEST with %s and %s", api, oclass)
                     self.container = self.add_containers(oclass)
                     try:
-                        self.execute_mdtest()
+                        self.execute_mdtest_with_flags(mdtest_np, mdtest_ppn)
```

Comment:
Similar to IOR, I don't think we should add more functions like this, since they tend to blow up

Reply:
- [ ] Remove useless method `execute_mdtest_with_flags()`